pages/api/upload/index.js

```diff
@@ -22,13 +22,17 @@ export default async function handler(request, response) {
 
   const [fields, files] = await form.parse(request);
 
-  const file = files.cover[0];
-  const { newFilename, filepath } = file;
+  const images = [];
+  for (const file of files.cover) {
+    const { filepath, newFilename } = file;
 
-  const result = await cloudinary.v2.uploader.upload(filepath, {
-    public_id: newFilename,
-    folder: "nf",
-  });
+    const result = await cloudinary.v2.uploader.upload(filepath, {
+      public_id: newFilename,
+      folder: "nf",
+    });
 
-  response.status(200).json(result);
+    images.push(result);
+  }
```

Comment:
We could also use the `files.cover.map` approach plus `await Promise.all()` instead of using a for loop. I don't know which one is more suitable for the students, probably it does not matter much.

Reply:
You are right, that would be another great option here. Since I also think that it does not matter too much, I would leave it as it is for the moment and in case students struggle a lot with this part of the code, we could either add a comment for explanation or change it to the `Promise.all` approach